pkg/plugins/runtime/k8s/controllers/gatewayapi/gateway_controller.go

```diff
@@ -95,6 +100,21 @@ func (r *GatewayReconciler) Reconcile(ctx context.Context, req kube_ctrl.Request
 	return kube_ctrl.Result{}, nil
 }
 
+func (r *GatewayReconciler) setKumaOwner(ctx context.Context, gateway *gatewayapi.Gateway) error {
```

Comment:
I think we need a new name for this. It is setting the owner of a Kuma Gateway object, so the name should make that clear to the reader.